src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -129,16 +129,19 @@ public boolean isPersonInLineup(Person person, Lineup lineup) {
     public void deletePersonFromLineup(Person person, Lineup lineup) {
         lineup.removePlayer(person);
         person.removeFromLineup(lineup);
+        addressBook.refresh();
```

Comment:
Do you think it would be better if we refresh after each command is executed to avoid repetitive calling of this method?

Reply:
Noted. I have updated the methods!